templates/partials/access_point_input.html

```diff
@@ -6,6 +6,8 @@
     data-source_url="{{ source_url }}"
     data-source_detail_url="{% url 'view-source' source_id %}"
     data-source_publication_country="{{ publication_country|default_if_none:"" }}"
+    data-published_on="{{ published_on|default_if_none:"" }}"
```

Comment:
Let's give `published_on` a more sensible default:

```suggestion
    data-published_on="{{ published_on|default:"Publication date unavailable" }}"
```
(Seems like `published_on` can be the empty string when it's null, so I had to use `default` instead of `default_if_none` to get it to work.)

This will help improve formatting in cases where the publication date is unavailable, such as http://localhost:8000/en/person/edit/089ea537-b531-472f-bf2a-6896e4a0e43c/:

![Screen Shot 2019-08-14 at 9 45 51 AM](https://user-images.githubusercontent.com/14170650/63030717-69175300-be78-11e9-8276-4c9b8eff0a54.png)